resources/assets/pages/ShowPost.js

```diff
@@ -31,6 +31,7 @@ const SHOW_POST_QUERY = gql`
 const ShowPost = () => {
   const { id } = useParams();
   const title = `Post #${id}`;
+  document.title = title;
```

Comment:
Also wanted to ask why we have to re-assign this here?

Reply:
The default document title (the `<title>` tag in our `<head>` HTML) gets returned as "Rogue" when it's isn't set in any React component. It'd be sweet to add this to all of our Page components to keep it consistent, but aim to make the changes in future cleanup PR.
 
<img width="463" alt="Screen Shot 2019-12-06 at 12 03 28 PM" src="https://user-images.githubusercontent.com/1236811/70352644-7047df80-1820-11ea-86c5-54ff27b942cb.png">

Reply:
ohhhhhhhhhh yes that's an awesome addition 💯